tests/test_ref.py

```diff
@@ -650,3 +656,12 @@ def test_change_spec(self, spec):
         with pytest.raises(KeyError):
             after[270]
         assert list(after.find_all(270)) == [[268, 0, 270], [268, 1, 270]]
+
```

Comment:
I guess you made the change based on an older version of pyfixmsg, maybe you could rebase to the latest. I found the test function **test_from_dict_with_none()** is not found, also 4 lines are missing in function **test_update_with_none()**.

Reply:
I’ll check, I didn’t notice them missing in the review on the pull request..

Reply:
I don't see changes in the public repo for these -- could these be changes that are available in your private repo?

Reply:
As I checked again you don't need to rebase, because I got a new edition of code, sorry for that,